blocks/grid/grid.editor.js

```diff
@@ -0,0 +1,215 @@
+export default function config() {
+  return {
+    inline: {
+      component: 'InlineEditGridComponent',
+      inputs: [],
+    },
+    attributes: {
+      grid: {
+        'template-rows': {
+          type: 'text',
+          label: 'Row',
+          helpText: 'The row number.',
+          value: '1fr',
+        },
+        'template-columns': {
+          type: 'text',
+          label: 'Columns',
+          helpText: 'The column number.',
+          value: '1fr 1fr',
+        },
+        gap: {
+          type: 'text',
+          label: 'Gap',
+          helpText: 'The gap between the grid items.',
+          value: '20px',
+        },
+      },
+      data: {
+        level: {
+          type: 'text',
+          label: 'Level',
+          helpText: 'The level of the grid.',
+          value: '1',
+        },
+        height: {
+          type: 'text',
+          label: 'Height',
+          helpText: 'The height of the grid.',
+          value: '100%',
```

Comment:
default height should be based on content, this should be left unset or `initial`